.github/workflows/integration.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Set up Python
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v4
```

Comment:
why was that change necessary?

Reply:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#patching-your-actions-and-workflows
and
https://github.com/actions/setup-python/releases/tag/v4.3.0
(there is a comment in the git-commit for this change)